README.md

```diff
@@ -168,13 +169,9 @@ Should be compatible with all robots and control boxes with the newest firmware.
 
 ### Tested with:
 
-* Real UR10 with CB2 running 1.8.14035
-* Real UR5 with CB2 running 1.8.14035
-* Simulated UR3 running 3.1.18024
-* Simulated UR5 running 3.0.16471
-* Simulated UR5 running 1.8.16941
-* Simulated UR5 running 1.7.10857
-* Simulated UR5 running 1.6.08725
+*Real UR5 and UR3 with 3.5.1.1.10661 (Dec 13 2017)
+
+*Real UR10 with CB2 running 1.8.14035
```

Comment:
Where are these changes coming from? The new version with just two lines doesn't match the previous, more detailed, version.

Reply:
I don't know why the lines were deleted either; maybe something went wrong with the rebase. I only remember adding line 172. I updated this and included our newer robot.

Reply:
Probably rebasing shenanigans, yes. :+1: